packages/rulesets/src/spectral/functions/consistent-put-response-body.ts

```diff
@@ -0,0 +1,28 @@
+// Verify that put response body schemas are consistent between create and replace.
+
+// This function assumes it is running on a resolved doc.
+const consistentputresponsebody = (pathItem: any, _opts: any, paths: any) => {
+  if (pathItem === null || typeof pathItem !== "object") {
+    return []
+  }
+  const path = paths.path || []
+
+  const errors: any = []
+
+  // resource schema is create operation response schema
+  const createResponseSchema = (op: any) => op?.responses?.["201"]?.schema
+  const resourceSchema = createResponseSchema(pathItem.put) || createResponseSchema(pathItem.patch)
```

Comment:
>createResponseSchema(pathItem.patch) [](http://example.com/codeflow?start=63&length=36)

can we add this to the documentation too?
that patch 201 response can/also needs to be the same?

Reply:
this was unintentionally added as a part of a copy paste. Will remove it.